.eslintrc

```diff
@@ -0,0 +1,52 @@
+{
+  "extends": [
+    "lisk-base",
+    "airbnb-base"
+  ],
+  "plugins": [
+    "import"
+  ],
+  "globals": {
+    "describe": true,
+    "it": true,
+    "beforeEach": true,
+    "afterEach": true,
+    "ipc": true,
+    "PRODUCTION": true,
+    "TEST": true
+  },
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "rules": {
+    "func-names": "off",
+    "global-require": "off",
+    "new-cap": ["error", {
+      "newIsCapExceptions": ["mnemonic"],
+      "capIsNewExceptions" : [
+        "When",
+        "Then",
+        "Given",
+        "After",
+        "Before"
+      ]
+    }],
+
+    "no-loop-func": "off",
+    "no-plusplus": "off",
+    "no-restricted-properties": "off",
+    "no-return-assign": "off",
+    "no-underscore-dangle": "off",
+    "import/no-extraneous-dependencies": ["error", 
+      {
+      "devDependencies": [
+        "./src/**/*.test.js",
+        "./features/*/*.js",
+        "./src/**/stories.js"
+        ]
+      }
+    ],
+    "no-param-reassign": "off"
+  }
```

Comment:
While using angular, you pass all the dependencies to your controllers and directives and then sometimes they need reassignment. since the aim of these 3 issue is not to rewrite the entire code, I have to disable this rule.